extensions/amp-form/0.1/amp-form.js

```diff
@@ -192,7 +197,10 @@ function installSubmissionHandlers(win) {
 function installAmpForm(win) {
   return getService(win, 'amp-form', () => {
     if (isExperimentOn(win, TAG)) {
-      installStyles(win.document, CSS, () => installSubmissionHandlers(win));
+      installStyles(win.document, CSS, () => {
+        installSubmissionHandlers(win);
+        actionServiceFor(win).addEvent('submit');
```

Comment:
Cool thanks. Just did this it works great. 

Does the AMP `on=event:object.method` syntax allows for listening for multiple actions? I wasn't sure for example how to make an example that listen to both `submit-success` and `submit-error`. Is this not currently supported, if not do we have an open bug to introduce that?